src/wp-includes/theme.php

```diff
@@ -2612,11 +2612,18 @@ function get_theme_starter_content() {
  * @since 5.3.0 The `html5` feature now also accepts 'script' and 'style'.
  * @since 5.3.0 Formalized the existing and already documented `...$args` parameter
  *              by adding it to the function signature.
+ * @since 5.4.0 The `disable-custom-gradients` feature limites to default gradients or gradients added
+ *              through `editor-gradient-presets` theme support.
  * @since 5.5.0 The `core-block-patterns` feature was added and is enabled by default.
  * @since 5.5.0 The `custom-logo` feature now also accepts 'unlink-homepage-logo'.
  * @since 5.6.0 The `post-formats` feature warns if no array is passed as the second parameter.
  * @since 5.8.0 The `widgets-block-editor` feature enables the Widgets block editor.
+ * @since 5.8.0 The `block-template-parts` feature allow to edit any reusable template part from site editor.
  * @since 6.0.0 The `html5` feature warns if no array is passed as the second parameter.
+ * @since 6.1.0 The `disable-layout-styles` feature disables the default layout styles.
+ * @since 6.1.0 The `link-color` feature allows to enable the link color setting.
+ * @since 6.3.0 The `border` feature allows themes without theme.json to add border styles to blocks.
+ * @since 6.4.0 The `block-templates` feature allow to add editable block templates to post or page.
```

Comment:
```suggestion
 * @since 6.4.0 The `block-templates` feature allows to add editable block templates to post or page.
```

Reply:
The description for this feature is "Whether a theme uses block-based templates", so this is probably better:

```suggestion
 * @since 6.4.0 The `block-templates` feature indicates whether a theme uses block-based templates.
```

Reply:
Also, are you sure 6.4.0 is correct for this? `create_initial_theme_features` says it was added in 6.0.1

Reply:
`block-templates` was added at `5.8.0` as experimental according to this blog and then eventually updated.
So could you confirm which version should I mention in `@since`?
https://fullsiteediting.com/lessons/adding-full-site-editing-features-to-classic-themes/#h-4-add-editable-block-templates-for-posts-and-pages
@swissspidy

Reply:
According to https://core.trac.wordpress.org/ticket/53176 it was 5.8.0 indeed. So this needs to be 5.8.0.